flume-ng-sinks/flume-dataset-sink/pom.xml

```diff
@@ -73,12 +73,6 @@ limitations under the License.
       <artifactId>avro</artifactId>
     </dependency>
 
-    <dependency>
```

Comment:
Removed this dependency because:
- it seems to be not needed
- it brings org/apache/avro/** classes and clashes with the explicit dependency above